src/app/profile/invite/invite-item/invite-item.component.html

```diff
@@ -38,7 +38,7 @@ <h3>{{ item.register_nick }}</h3>
 <ng-template #notregistered>
   <ng-container *ngIf="item.sent; else notsent">
     <div class="detail-label">{{ 'Versendet am' | translate }} {{ item.sent | shortdate }}</div>
-    <h3 (click)="resendInvite()" style="cursor: pointer">{{ 'Erneut senden' | translate }}</h3>
+    <h2 (click)="resendInvite()" style="cursor: pointer">{{ 'Erneut senden' | translate }}</h2>
```

Comment:
In the design this looks the same size as `item.register_nick` but non-bold and should not be upper-cased.